BigQuery/tests/System/ManageTablesTest.php

```diff
@@ -347,4 +347,93 @@ private function runJob($jobConfig, $client = null)
 
         $this->assertArrayNotHasKey('errorResult', $job->info()['status']);
     }
+
+    /**
+     * @dataProvider referenceFileSchemaTestUris
+     */
+    public function testCreateTableWithReferenceFileSchemaUri($sourceUris, $refUri, $format)
+    {
+        $id = uniqid(self::TESTING_PREFIX);
+        $options = [
+            'externalDataConfiguration' => [
+                'sourceUris' => $sourceUris,
+                'referenceFileSchemaUri' => $refUri,
```

Comment:
There are plans for adding samples for creating an external table with a reference schema file ? I'm asking because with the Load job at least with have a reference to the feature with the `referenceFileSchemaUri` method in the `LoadJobConfiguration.php`, but that doesn't exist on the `createTable` method.

Reply:
You are right, it is difficult for the user to find it. I did not find any sample tags for this. I have mentioned it in the design doc as well.

Reply:
We will discuss it in the design doc(started a thread there). If we get a tag, will make sample for it. Thank you for mentioning it.